modules/nodes-descheduler-strategies.adoc

```diff
@@ -20,6 +20,8 @@ Duplicate Pods::
 The `RemoveDuplicates` strategy ensures that there is only one Pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate Pods are evicted for better spreading of Pods in a cluster.
 +
 This situation could occur after a node failure, when a Pod is moved to another node, leading to more than one Pod associated with a ReplicaSet, ReplicationController, Deployment, or Job on that node. After the failed node is ready again, this strategy evicts the duplicate Pod.
++
+This strategy has an optional parameter, `ExcludeOwnerKinds`, that allows you to specify a list of `Kind` types. If a Pod has any of these types listed as an `OwnerRef`, then that Pod is not considered for eviction.
```

Comment:
omit `then` after the comma

>! if...then _conj_
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”

Reply:
This is a tough habit for me to break! :laughing: